transport/grpc/options.go

```diff
@@ -119,25 +121,35 @@ func ClientMaxSendMsgSize(clientMaxSendMsgSize int) TransportOption {
 	}
 }
 
-// ClientTLS says to use TLS with the system certificate pool on the client side.
-//
-// The default is to not use TLS.
-func ClientTLS() TransportOption {
-	return func(transportOptions *transportOptions) {
-		transportOptions.clientTLS = true
-	}
-}
```

Comment:
Assume that this code was never released - otherwise this is breaking.

Reply:
True - this was never released.